services/core-web/src/components/Forms/ExplosivesPermit/MagazineFormNew.tsx

```diff
@@ -0,0 +1,308 @@
+import React from "react";
```

Comment:
Rather than add a bunch of feature flags, I opted to create "New" files that can be loaded conditionally.

Reply:
Apologies that it doesn't show a proper diff as a result.  Most of the changes are style related.